ui-tests/src/test/java/org/openmrs/reference/LoginTest.java

```diff
@@ -34,45 +38,44 @@ public void verifyModulesAvailableOnHomePage() throws Exception {
         assertTrue(homePage.isConfigureMetadataAppPresent());
         assertTrue(homePage.isSystemAdministrationAppPresent());
     }
-
+    
```

Comment:
this too needs to be deleted

Reply:
Resolved thanks

Reply:
i still found the space, use your IDE to always see changes before committing, you can also use `git diff`